Services/Administration/OJS.Services.Administration.Data/IGridDataService.cs

```diff
@@ -10,7 +10,7 @@
 using System.Threading.Tasks;
 
 public interface IGridDataService<TEntity> : IService
-    where TEntity : class, IEntity
```

Comment:
Constraint

Reply:
Because UserProfile does not comes from IEntity<Tid>, therefore cannot be used in our workflow.

Reply:
Fixed